### PR TITLE
Add Hi.Events Software

### PR DESCRIPTION
**Pull request by daveearley**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/hi-events.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  - AGPL-3.0
+platforms:
+  - JavaScript
+  - Nodejs
+  - PHP
+  - Docker
+tags:
+  - Booking and Scheduling
+  - E-commerce
+depends_3rdparty: false
```
Comment (kokomo123) [inline software/hi-events.yml:15]:
```suggestion
```
This is already set to false, no need for this line 👍🏻

File: software/hi-events.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+name: "Hi.Events"
+website_url: "https://hi.events"
+source_code_url: "https://github.com/HiEventsDev/hi.events"
+description: "An open-source, feature-packed event management and ticketing platform for conferences, concerts, and more. Offering customizable event pages and embeddable ticket widgets."
+licenses:
+  - AGPL-3.0
+platforms:
+  - JavaScript
```
Comment (kokomo123) [inline software/hi-events.yml:8]:
```suggestion
```
No need for javascript since Nodejs is already there.

File: software/hi-events.yml (hunk 1 above)
Comment (daveearley) [reply software/hi-events.yml]: Updated 👍

**Review by kokomo123 — APPROVED** [review]
LGTM, No issues here. Pending other thoughts from collaborators, thank you for your contribution!

File: software/hi-events.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+name: "Hi.Events"
+website_url: "https://hi.events"
+source_code_url: "https://github.com/HiEventsDev/hi.events"
+description: "An open-source, feature-packed event management and ticketing platform for conferences, concerts, and more. Offering customizable event pages and embeddable ticket widgets."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Nodejs
+  - PHP
+  - Docker
```
Comment (Ki-er) [inline software/hi-events.yml:10]: > https://hi.events/docs/getting-started/deploying
```suggestion
platforms:
  - Docker
```

File: software/hi-events.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+name: "Hi.Events"
+website_url: "https://hi.events"
+source_code_url: "https://github.com/HiEventsDev/hi.events"
+description: "An open-source, feature-packed event management and ticketing platform for conferences, concerts, and more. Offering customizable event pages and embeddable ticket widgets."
```
Comment (Ki-er) [inline software/hi-events.yml:4]:
```suggestion
description: "Event management and ticketing platform for conferences, concerts, and more. Offering customizable event pages and embeddable ticket widgets."
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
Recommended changes.

**Review by nodiscc — APPROVED** [review]

**Pull request merged**